web-app/src/app/interface/RemoteConfig.ts

```diff
@@ -0,0 +1,10 @@
+import { remoteConfig } from '../../firebase';
+export type RemoteConfigValues = Record<string, string | number | boolean>;
+
+// Add default values for remote config here
+export const defaultRemoteConfigValues: RemoteConfigValues = {
+  enableGoogleSSO: true,
+  enableGithubSSO: true,
```

Comment:
We can remove these flags used for testing and add _enableAppleSSO_ as it will be used right after this PR is merged

Reply:
✅